grid/chat/channels/profile.supr

```diff
@@ -0,0 +1,191 @@
+name = "chat/channels"
+version = "1.0.0"
+
+"""
+Get Channels
+Retrieve paginated list of channels, e.g. text channels in slack or discord
+"""
+usecase GetChannels {
+  input {
+    """
+    Server
+    Representation of a server. Can be guild or server, depending on the provider capability.
+    """
+    server string
+
+    """
+    Types
+    List of categories of channels
+    """
+    types [
+      enum {
+        private
+        public
+      }
+    ]
+
+    """
+    Limit
+    Maximum number of channels to retrieve
+    """
+    limit number
+
+    """
+    Page
+    Identification of page with paginated results (cursor)
+    """
+    page string
+  }
+
+  result {
+    """
+    Channels
+    List of retrieved channels
+    """
+    channels! [Channel]
+
+    """
+    Next Page
+    Cursor to the next page for pagination forwards
+    """
+    nextPage string
+
+    """
+    Rate Limit
+    Rate limit details
+    """
+    rateLimit RateLimit
+  }
+
+  error Error
+
+  example Successful {
+    input {
+      types = ['public'],
+      limit = 2
+    }
+
+    result {
+      channels = [
+        {
+          id = '1'
+          createdAt = 1546167104
+          name = 'random'
+          numOfMembers = 100
+        }
+        {
+          id = '2'
+          createdAt = 1546167104
+          numOfMembers = 100
+        }
+      ]
+    }
+  }
+
+  example Failed {
+    input {
+      server = 'not-existing-server'
+    }
+
+    error {
+      title = 'server_not_found'
+    }
+  }
+}
+
+model Channel {
+  """
+  Id
+  Unique identifier of the channel
+  """
+  id! string!
+
+  """
+  Created At
+  UNIX timestamp (e.g. 1095379198.75) when the channel was created
+  """
+  createdAt! number!
+
+  """
+  Name
+  Name of retrieved channel
+  """
+  name string
+
+  """
+  Description
+  Description of the channel. Can describe purpose or use of the channel.
+  """
+  description string
+
+  """
+  Num Of Member
+  Number of members in retrieved channel
+  """
+  numOfMembers number
```

Comment:
I'd avoid shortening in fields' names as much as possible. I suggest renaming this to `membersCount`

```suggestion
  """
  Members Count
  Number of members of the retrieved channel
  """
  memebersCount number
```